MicroProfile-JWT-Auth/tck-runner/pom.xml

```diff
@@ -63,6 +63,11 @@
                     <groupId>fish.payara.microprofile.jwt-auth</groupId>
                     <artifactId>microprofile-jwt-auth</artifactId>
                 </exclusion>
+                 <!-- exclude dependency inheritance for server-managed from tck extension -->
+                <exclusion>
+                    <groupId>fish.payara.arquillian</groupId>
```

Comment:
Weren't we going to look into why this is happening? TCK extension only relies on some Jersey client stuff and Json.